src/crossChain/AxelarProvider.ts

```diff
@@ -84,6 +86,112 @@ export class AxelarProvider implements CrossChainProvider {
     return tx.transactionHash
   }
 
+  async getRegisterNameRoute(
+    getRegisterNameCrossChainData: RegisterNameCrossChainData
+  ): Promise<RouteResponse> {
+    if (!this.squid.initialized) {
+      await this.init()
+    }
+    const {
+      fromAddress,
+      fromAmount,
+      fromChain,
+      fromToken,
+      toChain,
+      toAmount, // the registration price
+      enableExpress = true,
+      slippage = 1, // 1 is "normal" slippage. Always set to 1
+      name
+    } = getRegisterNameCrossChainData
+
+    const ERC20ContractInterface = new ethers.utils.Interface(ERC20)
+    const destinationChainMANA = getContract(ContractName.MANAToken, toChain)
+      .address
+    const controllerContract = getContract(
+      ContractName.DCLControllerV2,
+      toChain
+    )
+    const ControllerV2Interface = new ethers.utils.Interface(DCLControllerV2)
+
+    return this.squid.getRoute({
+      fromAddress,
+      fromAmount,
+      fromToken,
+      fromChain: fromChain.toString(),
+      toToken: destinationChainMANA,
+      toChain: toChain.toString(),
+      toAddress: controllerContract.address,
+      enableBoost: enableExpress,
+      slippageConfig: {
+        autoMode: slippage
+      },
+      postHook: {
+        description: '',
+        chainType: ChainType.EVM,
+        fundAmount: '1',
+        fundToken: destinationChainMANA,
+        calls: [
+          // ===================================
+          // Approve MANA to be spent by Decentraland contract
+          // ===================================
+          {
+            chainType: ChainType.EVM,
+            callType: SquidCallType.DEFAULT,
+            target: destinationChainMANA,
+            value: '0',
+            callData: ERC20ContractInterface.encodeFunctionData('approve', [
+              controllerContract.address,
+              toAmount
+            ]),
+            payload: {
+              tokenAddress: NATIVE_TOKEN,
+              inputPos: 0
+            },
+            estimatedGas: '50000'
+          },
```

Comment:
We should add the cases in which:
- The user has already approved an infinite amount in the future and don't provide this call
- The user has approved some amount which is higher than this so there is no need to make the call
- The user has approve some amount which is lower, meaning a call to allow 0 has to be made before another call to approve the amount has to be made